v4l2: use proper headers for OpenBSD's V4L2 support
c7905ecc665b1b464bbf13ddc395e2e7f8ddd8b2

Index: modules/access/v4l2/v4l2.h
--- modules/access/v4l2/v4l2.h.orig
+++ modules/access/v4l2/v4l2.h
@@ -18,7 +18,12 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston MA 02110-1301, USA.
  *****************************************************************************/
 
-#include <linux/videodev2.h>
+#if defined(HAVE_LINUX_VIDEODEV2_H)
+#   include <linux/videodev2.h>
+#elif defined(HAVE_SYS_VIDEOIO_H)
+#   include <sys/ioccom.h>
+#   include <sys/videoio.h>
+#endif
 
 /* libv4l2 functions */
 extern int (*v4l2_fd_open) (int, int);
